zcash_client_sqlite/src/wallet/sapling.rs

```diff
@@ -126,6 +126,28 @@ fn to_spendable_note(row: &Row) -> Result<ReceivedSaplingNote<ReceivedNoteId>, S
     })
 }
 
+/// Utility method for determining whether we have any spendable notes
+///
+/// If the tip shard has unscanned ranges below the anchor height and greater than or equal to
+/// the wallet birthday, none of our notes can be spent because we cannot construct witnesses at
```

Comment:
This function does not query or take as a parameter the wallet birthday. Why is it referenced here?

Reply:
This is where I *thought* a bug might be, but `v_sapling_shard_unscanned_ranges` is what is actually bounding on the wallet birthday. I'll add a comment in the body to that effect.